charts/astronomer/values.yaml

```diff
@@ -180,6 +180,17 @@ houston:
     # Default here is to run at midnight every night
     schedule: "0 0 * * *"
 
+  # Check for Airflow Updates
+  # This runs as a CronJob
+  updateAirflowCheck:
+    # Enable check updates CronJob
+    enabled: true
+
+    # url for update service
+    url: https://updates.astronomer.io/astronomer-certified
+
+    # Default here is to run at midnight every night
+    schedule: "0 0 * * *"
```

Comment:
```suggestion
    schedule: "{{ div (randNumeric 2) 2 }} 0 * * *"
```

Reply:
@danielhoherd what do you think just change to "1 0 * * *" ?

Reply:
That is not going to spread the request load out over time. If we don't pick a random time, all installed hosts will request the resource at the same time, potentially overloading the server. However, after talking to Vishwas and looking at the number of clients we have it's not really a big deal right now.